pages/draft/2020-12/release-notes.md

```diff
@@ -155,8 +161,9 @@ Here's how you would covert a schema using `$recursiveRef` to use `$dynamicRef`.
   </tr>
   <tr>
     <td>
+          tree schema extensible
```

Comment:
This isn't what I mean.  I believe the code block itself supports a title.

@jdesrosiers I can't find the format for this.  Can you help, please?

Reply:
I explored the option for adding titles to code blocks but couldn't find a standard method that Markdown supports. I'm open to any specific examples or guidelines you might have.

Reply:
This is something specific that we've implemented, but I can't find the examples I'm thinking of.  Jason is more familiar with it.

Reply:
I implemented support for code block titles for the markdown in the spec repo. I'm not aware of what might be supported in the website repo.

This was the syntax, but I don't know if something like this is supported on the website.
```
```json "JavaScript with a title"
{ "foo": 42 }
```

Reply:
Okay.  Thanks @jdesrosiers.

@jayprakash25 let's keep what you have, then.  Is there a way you can fix the unevenness between the columns shown in the screenshot?

Reply:
> Okay. Thanks @jdesrosiers.
> 
> @jayprakash25 let's keep what you have, then. Is there a way you can fix the unevenness between the columns shown in the screenshot?

@gregsdennis 

> Okay. Thanks @jdesrosiers.
> 
> @jayprakash25 let's keep what you have, then. Is there a way you can fix the unevenness between the columns shown in the screenshot?

![Macbook Pro-1709085730829](https://github.com/json-schema-org/website/assets/110524709/8b2cd36a-5f27-40cb-b49c-8b1223588973)

Yes, I have done that. The uneveneness between columns is fixed now. I have updated the pr.